### PR TITLE
Init

### PR DESCRIPTION
**Pull request by l0g1c-80m8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+### Install Oculus ###
+- Perform the initial setup of your VR headset. For oculus/quest see [this](https://www.meta.com/blog/quest/you-got-a-quest-2-heres-how-to-set-it-up/).
+- Set the VR headset to run with SteamVR. For oculus/quest see [this](https://docs.varwin.com/latest/en/instructions-for-using-the-oculus-quest-2-headset-2260861409.html).
+
+### Set up the Repositories ###
+Clone the project using: 
+```
+git clone git@github.com/SophieHsu/view-aware-hrc.git
+or
+git clone https://github.com/SophieHsu/3d-plan-eval
```
Comment (SophieHsu) [inline README.md]:
```suggestion
git clone https://github.com/SophieHsu/3d-plan-eval --recursive
```

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+or
+./setup.bash # for linux/mac
+```
+
+## Usage ##
+
+### Running the scripts ###
+Launch the planner and the VR kitchen environment using the following command.
+```
+python3 launch.py
```
Comment (SophieHsu) [inline README.md:65]: Do the users need to conda beforehand?

File: vr_kitchen
Comment (SophieHsu) [inline vr_kitchen:1]: Would it be better to link the version we are going to finish today?

File: launch.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            # "--practice",  # Flag to indicate whether this is a practice session
+        ]
+    ),
+    (  # fov_aware_planner
+        # venv path
+        os.path.join(__current_dir_path, "fov_aware_planner", "venv"),
+        # script directory
+        os.path.join(__current_dir_path, "fov_aware_planner", "overcooked_ai_py"),
+        # script path
+        "steak_api_test.py",
```
Comment (SophieHsu) [inline launch.py:29]: Let's also include the restart mechanism as the default option here.

**Review by SophieHsu — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (l0g1c-80m8) [reply README.md:65]: No. Setup takes care of that.

File: launch.py (hunk 3 above)
Comment (l0g1c-80m8) [reply launch.py:29]: Not tested yet. Will do in another PR.

**Pull request merged**